confmap/internal/e2e/go.mod

```diff
@@ -4,13 +4,13 @@ go 1.21.0
 
 require (
 	github.com/stretchr/testify v1.9.0
-	go.opentelemetry.io/collector/confmap v0.106.0
+	go.opentelemetry.io/collector/confmap v0.106.1
 	go.opentelemetry.io/collector/confmap/converter/expandconverter v0.105.0
```

Comment:
should these have been updated?

Reply:
Seems even on mainline those are not updated as part of the release, but rather renovbot https://github.com/open-telemetry/opentelemetry-collector/pull/10743